server/services/GameLogic.js

```diff
@@ -210,4 +246,10 @@ module.exports = class GameLogic {
         this.roundNum = 0;
         this.players = [];
     }
+
+    /* Clear timeouts */
+    clearTimeouts() {
```

Comment:
Any other timeouts that you have created need to be reset in this function e.g. `nextRoundTimeout`